src/components/shared/footerStyles.tsx

```diff
@@ -0,0 +1,102 @@
+import { css } from '@emotion/core';
+import { Footer } from '@guardian/src-footer';
+import { brandBorder, brandText, space } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
+import { bool } from 'aws-sdk/clients/signer';
+import React from 'react';
+
+
+// Footer content styles 
+const container = css`
+		border-style: solid;
+		border-color: ${brandBorder.primary};
+		border-width: 0 1px 0 1px;
+	
+`
+
+const para = css`
+	${textSans.medium({ lineHeight: "tight" })};
+	max-width: 58.75rem;
+`
+
+const ul = css`
+	list-style: none;
+	padding: 0;
+	border-top: 1px solid ${brandBorder.primary};
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	grid-template-areas:
+		"link1 link3"
+		"link2 link4";
+`
+
+const li = css`
+	padding: ${space[1]}px 0 ${space[4]}px 0;
+	border-style: solid;
+	border-color: ${brandBorder.primary};
+	border-width: 0;
+`
+const anchor = css`
+	${textSans.medium({ lineHeight: "regular" })};
+	color: ${brandText.anchorPrimary};
+	text-decoration: none;
+`
+
+const link1 = css`
+	grid-area: link1;
+	border-right-width: 1px;
+`
+const link2 = css`
+	grid-area: link2;
+	border-right-width: 1px;
+`
+
+const CcpaListItem = (val: bool) => {
+    if (val){
+        return (
+            <li css={[li, link2]}>
+				<a  id='testing' css={anchor} href="https://www.theguardian.com/ccpaoptedin">
+					CCPA
+				</a>
+			</li>
+        )
+    } else {
+        return null;
+    }
+}
+
+
+export const footerContents = (ccpabool: boolean): JSX.Element=>{
+    console.log(`>>---.>>  ${ccpabool}`);
+    
+    return (
+        <div css={container}>
+            <p css={para}>
+                <strong>
+                    Promotion terms and conditions Offer subject to availability.
```

Comment:
Is it possible for these to get outdated? I thought this would come from design system?

Reply:
This was taken from the styling and content within source https://github.com/guardian/source/blob/4d2d6089af6423ad60cf19d1849608fd9fda040c/src/core/components/footer/stories/with-children.tsx

Reply:
Looks like the promotion message might just be a dummy example? I don't think we have any promotions we need disclaimers for

Reply:
I will definitely liaise with @SiAdcock regarding this, and amend as necessary either with a place holder/prop or omit.

Reply:
@JamieB-gu @webb04 apparenlty this would not be a design consideration, perhaps i shall take this out?